### ISSUE TITLE
unable to open file (in target "PacketTunnel" in project "Potatso") (in target 'PacketTunnel')

### ISSUE THREAD

**Issue by OyAnony** [haxpor/Potatso#121]
Potatso/Pods/Pods/Target Support Files/Pods-PacketTunnel/Pods-PacketTunnel.debug.xcconfig: unable to open file (in target "PacketTunnel" in project "Potatso") (in target 'PacketTunnel')

**Comment by RyanXie123** [comment]
the same

**Comment by edsburke** [comment]
I have the same issue, how do you guys resolve this? 

@haxpor would you please take a look.

My Env is:
Xcode Version 10.1 (10B61)
MacOS 10.14
Pod: 1.5.3

I notice that there's diff in Podfile.lock after running the build sequence in README. Does this matter?
```
-COCOAPODS: 1.6.0.beta.1
+COCOAPODS: 1.5.3
```

Thanks.

**Comment by edsburke** [comment]
It turns out COCOAPODS: 1.6.0.beta.1 or above is needed. You can run the following to resolve the issue for example. You may need to uninstall cocoapods first.
```
gem install cocoapods --pre
```

**Comment by OyAnony** [comment]
I have a problem on ios11，Have you ever encountered it?
，
At 2018-11-25 09:32:34, "edsburke" <notifications@github.com> wrote:


It turns out COCOAPODS: 1.6.0.beta.1 or above is needed. You can run the following to resolve the issue for example. You may need to uninstall cocoapods first.

gem install cocoapods --pre

**Comment by edsburke** [comment]
@OyAnony No, I didn't. My setup is literally what you've seen in the post.

**Comment by OyAnony** [comment]
Oh, my use on the 10.3.3 iPhone, but there is no network after the link VPN, SS configuration is no problem.




| |
欧阳
邮箱：ouyang_refuel@163.com
|

签名由 网易邮箱大师 定制

On 11/26/2018 02:20, edsburke wrote:

@OyAnony No, I didn't. My setup is literally what you've seen in the post.
